e2e/various-suite/navigation.spec.ts

```diff
@@ -3,41 +3,58 @@ import { fromBaseUrl } from '../utils/support/url';
 
 describe('Docked Navigation', () => {
   beforeEach(() => {
+    // This is a breakpoint where the mega menu can be docked (and docked is the default state)
     cy.viewport(1280, 800);
+    cy.clearAllLocalStorage();
     e2e.flows.login(Cypress.env('USERNAME'), Cypress.env('PASSWORD'));
 
     cy.visit(fromBaseUrl('/'));
   });
 
-  it('should remain docked when reloading the page', () => {
-    // Expand, then dock the mega menu
-    cy.get('[aria-label="Open menu"]').click();
-    cy.get('[aria-label="Dock menu"]').click();
+  it('should remain un-docked when reloading the page', () => {
+    // Undock the menu
+    cy.get('[aria-label="Undock menu"]').click({ force: true });
 
-    e2e.components.NavMenu.Menu().should('be.visible');
+    e2e.components.NavMenu.Menu().should('not.exist');
 
     cy.reload();
+    e2e.components.NavMenu.Menu().should('not.exist');
+  });
+
+  it('Can re-dock after undock', () => {
+    // Undock the menu
+    cy.get('[aria-label="Undock menu"]').click({ force: true });
+    cy.get('[aria-label="Open menu"]').click({ force: true });
+    cy.get('[aria-label="Dock menu"]').click({ force: true });
+
```

Comment:
If do not add force: true. when I run e2e tests locally (yarn e2e:dev), in main or here I get 

`
Timed out retrying after 30050ms: cy.click() failed because this element is not visible:

<button id="dock-menu-button" variant="secondary" aria-label="Dock menu" class="css-duxynr" type="button" tabindex="0"></button>

This element <button#dock-menu-button.css-duxynr> is not visible because it has an effective width and height of: 0 x 0 pixels.
```

For me the icons are not loading in e2e tests. Does anyone else see this?